src/api/Comments.js

```diff
@@ -48,10 +48,18 @@ class Comments extends OffsetBasedAPI {
             return;
         }
 
-        // Manually create the entries array if creating/updating a comment
-        const entries = data.entries ? data.entries : [data];
+        // We don't have entries when updating/creating a comment
+        if (!data.entries) {
+            const comment = {
```

Comment:
Since the code here is similar/the same as the `map`below, why not extract into a util function, and reuse?
```
extractComment(data) {
    const { tagged_message, message }
     return {
        ...data,
        tagged_message: tagged_message !== '' ? tagged_message : message
    }
}